File: PR1/PR1_Ej2/BicicletaMontania.py
from Bicicleta import Bicicleta
import copy

class BicicletaMontania(Bicicleta):
    def __init__(self):
        self.tipo="Montania"

    def mostrarTipo(self):
        print("Soy una bici de ", self.tipo,"\n")

    def clone(self):
        return copy.deepcopy(self)

File: PR1/PR1_Ej1/src/BicicletaMontania.java
import java.util.Random;

public class BicicletaMontania extends Bicicleta{

    public BicicletaMontania(){
        Random rand = new Random();
        id = rand.nextInt(11);
        tipo = "montania";
    }
    @Override
    public void mostrarTipo() {
        System.out.println("Soy una bici de " + this.tipo + " con id : " + this.id + "\n");
    }
}
